string                      agent_name
string                      id
mac_ros_bridge/Position     destination
uint64                       deadline